t3/api/src/lib.rs

```diff
@@ -15,6 +15,19 @@ mod autogenerated_code {
     }
 }
 
+pub mod tonic {
+    pub mod external {
+        pub mod v1 {
+            tonic::include_proto!("external.v1");
```

Comment:
not sure on this nesting here, may need to wait until the actual tonic implementations of gRPC is done